src/data/faq.json

```diff
@@ -28,7 +28,19 @@
                             "As a media representative, please direct your enquiries to <a href='mailto:press@sap.com'>press@sap.com</a>.",
                             "The various feedback channels of the <a href='https://www.coronawarn.app/de/community/'>Corona-Warn-App-Community<a/> will help you with your questions about the open source project of the Corona-Warn-App."
                         ]
+                    },
+                    {
+                        "title": "How to reach us from abroad - International phone numbers",
+                        "anchor": "international_phone_numbers",
+                        "active": true,
+                        "textblock": [
+                            "From abroad you can reach us at the following international phone numbers:",
+                            "Technical hotline: +49 30 498 75401 available Monday to Saturday from 7 a.m. to 10 p.m. (except on national holidays)",
+                            "TAN hotline: +49 30 498 75402 available 24 hours a day, seven days a week",
+                            "Note: These numbers are <b>not free of charge</b> - the cost of the international call is based on the local contract with the user's provider."

```

Comment:
"the user's provider" -> "your provider"

Reply:
ok

Reply:
actually, remove also the word **local** ? ... based on the contract ...